frontend/src/bundles/common/components/stepper/stepper.tsx

```diff
@@ -0,0 +1,59 @@
+import { IconName } from '../../icons/icons.js';
+import { Box, Flex, Header, Icon, IconButton } from '../components.js';
+import { Step } from './components/components.js';
+import { getStepStatus } from './helpers/helpers.js';
+
+type Properties = {
+    steps: string[];
+    currentStep: string;
+};
+
+const stepsElements: React.FC<Properties> = ({ steps, currentStep }) => {
```

Comment:
Why the name of component doesn't start with capital letter?

Reply:
And actually, why do you need this component at all? Why can't you go through the steps array and create Step components inside Stepper directly? I think this component is unnecessary.

Reply:
done